added documentation for scoping agent pools to workspaces on agents page (#9)
hashicorp/terraform-docs-agents: website/docs/cloud-docs/agents/index.mdx

   ![Screenshot: Unable to delete an agent pool with attached workspace](/img/docs/agent-delete-invalid.png)

> ### Scope an Agent Pool to specific workspaces

Reviewer: ```suggestion
### Scope an Agent Pool to Specific Workspaces
```
```suggestion
### Scope an Agent Pool to specific workspaces
```

> 1. Navigate to **Organization Settings > Agents** and click on the name of the pool you would like to update.

Reviewer: ```suggestion

To scope an agent pool to specific workspaces within the organization:
1. Click **Settings** and then **Agents**. A list of agent pools for the organization appears.
1. Click the name of the pool you would like to update.
```

> 1. Click "Grant access to specific workspaces" option. This will open a new search menu for searching workspaces within this organization.
> 
> 1. Start typing the name of the workspace you would like to grant access to the agent pool and select it from the drop down list.
> 
> 1. Click "Save".

Reviewer: Can you select more than one workspace from the list at a time? 

```suggestion
1. Click **Grant access to specific workspaces**. A menu opens where you can search for workspaces within this organization.
1. Select the workspace you would like to grant access to the agent pool.
1. Click **Save**.
```
```suggestion
1. Click "Grant access to specific workspaces" option. This will open a new search menu for searching workspaces within this organization.

1. Start typing the name of the workspace you would like to grant access to the agent pool and select it from the drop down list.

1. Click "Save".
```

   ![Screenshot: Unable to delete an agent pool with attached workspace](/img/docs/agent-delete-invalid.png)

### Scope an Agent Pool to Specific Workspaces

> Scoping an agent pool lets you specify which workspaces are allowed to use it. The agent pool will then only be available from within the chosen workspaces; it will not appear other workspace's lists of available agent pools. Scoping an agent pool does not remove it from workspaces that are actively using it. To remove access, you must first remove the agent pool from within the workspace's settings.
> To scope an agent pool to specific workspaces within the organization:

Reviewer: ```suggestion
Scoping an agent pool lets you specify which workspaces are allowed to use it. The agent pool will then only be available from within the chosen workspaces; it will not appear other workspace's lists of available agent pools. Scoping an agent pool does not remove it from workspaces that are actively using it. To remove access, you must first remove the agent pool from within the workspace's settings.

To scope an agent pool to specific workspaces within the organization:
```
